cmd/lotus-miner/actor.go

```diff
@@ -1028,6 +1167,115 @@ var actorConfirmChangeWorker = &cli.Command{
 	},
 }
 
+var actorConfirmChangeBeneficiary = &cli.Command{
+	Name:      "confirm-change-beneficiary",
+	Usage:     "Confirm a beneficiary address change",
+	ArgsUsage: "[minerAddress]",
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "really-do-it",
+			Usage: "Actually send transaction performing the action",
+			Value: false,
+		},
+		&cli.BoolFlag{
+			Name:  "existingBeneficiary",
+			Usage: "send confirmation from the existing beneficiary address, defaults to the new beneficiary address",
```

Comment:
I personally think this is better as not defaulting -- I would have 2 flags `existing-beneficiary` and `new-beneficiary`, and exactly one of those 2 flags must be provided.